### ISSUE TITLE
What validation data is using?

### ISSUE THREAD

**Issue by katsu-77** [clovaai/voxceleb_trainer#168]
Training data : VoxCeleb2 
Test datas : VoxCeleb1 test set
Validation data : ?

What validation data is using ?
